elasticsearch/search-index-settings.json

```diff
@@ -0,0 +1,278 @@
+{
+  "settings": {
+    "index": {
+      "number_of_shards": 3,
```

Comment:
We should change the number of shards to 5 - there isn't a reason to move away from the default configuration